website/docs/kubernetes/kind.md

```diff
@@ -6,152 +6,76 @@ keywords: [podman desktop, podman, containers, migrating, kubernetes, kind]
-The stop command emits an error message, and the following Podman machine start seems to fail.
+#### Prerequisites
 
+* [You configured Podman](#configuring).
+* [You installed Kind](https://kind.sigs.k8s.io/).
 
 #### Procedure
 
-1. Stop all existing Kind clusters.
-2. Stop the Podman machine.
-3. Start the Podman machine.
-
-#### Workaround
-
-1. Keep existing Kind clusters running.
-2. Stop the Podman machine.
-3. Start the Podman machine.
-4. The Podman machine start reports success, but you cannot connect to the Podman machine.
-5. Stop the Podman machine.
-6. Start the Podman machine.
-
-#### Additional resources
-
-* [Kind](https://kind.sigs.k8s.io/)
-
-# Restarting your Kind clusters
-
-:::caution
-Kind cluster restart does not work in all environments. It is based on system features that may not be available on all
-configuration. Windows/WSL is an example of such configuration.
-:::
+* Delete the Kind cluster
 
-Kind does not provide start and stop command on Kind clusters. With Kind, you can only create and delete clusters. As
-delete will remove all the cluster configuration, users are not using it frequently.
-
-So there are some use cases where users may want to restart a Kind cluster. This is happening when the Podman system is stopped:
-in that case the container that was launched by the `kind create cluster` command may be switched to the stopped state
-but may not be removed so it may be restarted.
-
-## Identify stopped Kind clusters
-
-Kind clusters that are in the stopped state are reported by the `kind get clusters` command and their corresponding
-container is in the `Created` state. To list the containers for the Kind clusters, run the following command:
-
-<Tabs>
-  <TabItem value="Linux/MacOS" label="Linux/MacOS">
-
-  ```shell
-  podman ps -a | grep control-plane
-  ```
-  
-  </TabItem>
-  <TabItem value="Windows" label="Windows">
-
-  ```shell
-  podman ps -a | find "control-plane"
-  ```
-
-  </TabItem>
-</Tabs>
-
-If the container status is `Created` and not `Up...` then the Kind cluster is stopped and you can restart it with:
-
-  ```shell
-  podman start container-name
-  ```
-
-## Verification
-
-Before you check that the cluster is up and running using `kubectl`, you must first set the `kubectl` context to that
-cluster. So if you Kind cluster is called cluster_name, run the following command:
-
-  ```shell
-  kubectl config use-context kind-cluster_name
-  ```
-
-You can now run the following command to list all namespaces from the Kind cluster:
-
-  ```shell
-  kubectl get ns
-  ```
+   ```shell-session
+   $ kind delete cluster
+   ```
 
+## Restarting your Kind cluster {#restarting-kind}
 
+Kind has no command to restart a cluster.
 
+#### Workaround
 
+* Consider [deleting your Kind cluster](#kind-delete-cluster), and [creating a Kind cluster](#kind-create-cluster).
+* Consider replacing Kind by OpenShift Local.
```

Comment:
Maybe worth adding a link to https://developers.redhat.com/products/openshift-local ?